sentinelsat/scripts/cli.py

```diff
@@ -115,21 +114,26 @@ def search(
             raise ValueError('Cloud cover is only supported for Sentinel 2 and 3.')
         search_kwargs["cloudcoverpercentage"] = "[0 TO %s]" % cloud
 
-    # DEPRECATED: to be removed with next major release
-    elif sentinel2:
-        search_kwargs["platformname"] = "Sentinel-2"
-        logger.info('DEPRECATED: Please use --sentinel instead')
-
-    # DEPRECATED: to be removed with next major release
-    elif sentinel1:
-        search_kwargs["platformname"] = "Sentinel-1"
-        logger.info('DEPRECATED: Please use --sentinel instead')
-
     if query is not None:
         search_kwargs.update((x.split('=') for x in query.split(',')))
 
-    wkt = geojson_to_wkt(read_geojson(geojson))
-    products = api.query(wkt, start, end, order_by=order_by, limit=limit, **search_kwargs)
+    if geometry is not None:
+        wkt = geojson_to_wkt(read_geojson(geometry))
+
+    if uuid is not None:
+        uuid_list = [x.strip() for x in uuid.split(',')]
+        products = {}
+        for productid in uuid_list:
+            try:
+                products[productid] = api.get_product_odata(productid)
```

Comment:
`get_product_odata` will produce different set of attributes than `query`, which is a slight inconsistency when used with `--footprints`. This is a pretty minor use case, though, which we can safely ignore, though, I think.